units/provisioning_create_host.go

```diff
@@ -116,13 +116,17 @@ func (j *createHostJob) Run(ctx context.Context) {
 		}
 		if j.host == nil {
 			//host intent document has been removed by another evergreen process
+			notFoundError := errors.New("could not find host")
 			grip.Warning(message.Fields{
 				"host_id": j.HostID,
 				"task_id": j.TaskID,
 				"attempt": j.RetryInfo().CurrentAttempt,
 				"job":     j.ID(),
-				"message": "could not find host",
```

Comment:
We usually try to keep messages something readable; I think the only benefit to including the error in this grip statement is if we want the stack trace, otherwise wrapping it seems like clutter.
I do also think "could not find host" isn't a helpful user error to add to host create details at 127; do we know why the host isn't found at this point? Were we unable to create it?